Automations/TimeConverter/timeconverter.cpp

```diff
@@ -0,0 +1,85 @@
+#include "enum.cpp"
+#include <cstdio>
+#include <iostream>
+#include <string>
+#include <algorithm>
+#include <cctype>
+
+long double Convert(Time unit1, Time unit2, long double a)
+{
+    long double arr[8] = {60, 60, 24, 7, 30/7, 12, 10, 10};
+    long double temp = a;
+    if (unit1 >= unit2)
+    {
+        for (int i = unit2; i < unit1; i++)
+        {
+            temp *= arr[i];
+        }
+        return temp;
+    }
+    else if (unit1 < unit2)
+    {
+        for (int i = unit1; i < unit2; i++)
+        {
+            temp /= arr[i];
+        }
+        return temp;
+    }
+}
+
+int main()
+{
+    //Declaration
+    bool check1 = true;
+    bool check2 = true;
+    bool check3 = true;
+    std::string originalUnit;
+    std::string convertedUnit;
+    Time string1, string2;
+    long double time;
+    //Processing
+    std::cout << "This is the Time converter.\n";
+    while (check1)
+    {
+        //Ask for 1st unit of time
+        std::cout << "Please input the original unit of time that you want to convert.\n";
+        std::cout << "Valid inputs are: SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, YEAR, DECADE, CENTURY.\n";
+        std::cin >> originalUnit;
+        std::transform(originalUnit.begin(), originalUnit.end(),originalUnit.begin(), 
+                    [](unsigned char c) -> unsigned char { return std::toupper(c);});
+        string1 = convert(originalUnit);
```

Comment:
shouldn't the names of `string1` and `originalUnit` be interchanged ?

Reply:
Yes, I will fix it.